fluent-package/apt/systemd-test/update-to-next-version.sh

```diff
@@ -42,7 +42,8 @@ test $(eval $env_vars && echo $FLUENT_CONF) = "/etc/fluent/fluentd.conf"
 test $(eval $env_vars && echo $FLUENT_PACKAGE_LOG_FILE) = "/var/log/fluent/fluentd.log"
 test $(eval $env_vars && echo $FLUENT_PLUGIN) = "/etc/fluent/plugin"
 test $(eval $env_vars && echo $FLUENT_SOCKET) = "/var/run/fluent/fluentd.sock"
-test $(eval $env_vars && echo $FLUENT_PACKAGE_VERSION) = "$next_package_ver"
+# FLUENT_PACKAGE_VERSION will be updated after the next restart
+(! test $(eval $env_vars && echo $FLUENT_PACKAGE_VERSION) = "$next_package_ver")
```

Comment:
We should consider how to test this later, so it would be better to leave some TODO comments.

```suggestion
# FLUENT_PACKAGE_VERSION will be updated after the next restart
# TODO consider how to test the update of version info
(! test $(eval $env_vars && echo $FLUENT_PACKAGE_VERSION) = "$next_package_ver")
```